pkg/report/writer_test.go

```diff
@@ -317,6 +317,12 @@ func TestReportWriter_Template(t *testing.T) {
 			template:      `{{ toLower (getEnv "AWS_ACCOUNT_ID") }} {{ getCurrentTime }}`,
 			expected:      `123456789012 2020-08-10T07:28:17.000958601Z`,
 		},
+		{
+			name:          "happy path: replace colon with dash",
+			detectedVulns: []types.DetectedVulnerability{},
+			template:      `{{- $time := getCurrentTime -}} {{ toLower (getEnv "AWS_ACCOUNT_ID") }} {{ replaceAll $time ":" "-" }}`,
```

Comment:
Any reason to output `AWS_ACCOUNT_ID`? Looks like this test is irrelevant to AWS.

```suggestion
			template:      `{{- $time := getCurrentTime -}} {{ replaceAll $time ":" "-" }}`,
```